test/cpp/end2end/xds_end2end_test.cc

```diff
@@ -5442,16 +5443,36 @@ class XdsSecurityTest : public BasicTest {
       StartBackend(0);
       ResetBackendCounters();
       if (test_expects_failure) {
-        if (!SendRpc().ok()) break;
+        Status status = SendRpc();
```

Comment:
This local variable isn't really needed; just call `SendRpc()` directly in the `if` expression below.

Reply:
Done